File: Modul 7/NO 5/PRAK605-2210817310016-Riyo Aurora Guison.py
a = int(input())
MatriksA = []
print('Matriks A')
for i in range (a):
    k = list(map(int, input().split()))
    MatriksA.append(k)
MatriksB = []
print('Matriks B')
for i in range (a):
    k = list(map(int, input().split()))
    MatriksB.append(k)
MatriksAXB = []
print('Matriks AXB')
for i in range (a):
    MatriksAXB.append([])
    for j in range (a):
        y = 0
        for k in range (a):
            y = y + MatriksA[i][k] * MatriksB[k][j]
        MatriksAXB[i].append(y)
for baris in MatriksAXB:
    for k in baris:
        print(k, end=' ')
    print()

File: Modul 7/NO 5/PRAK605-2210817310016-Riyo Aurora Guison.c
#include<stdio.h>
int main()
{
    int MatriksA[10][10],MatriksB[10][10],MatriksAXB[10][10];
    int a,b,c,d,hasil=0;
    scanf("%d",&d);
    printf("Matriks A\n");
    for(a=0;a<d;a ++){
        for(b=0;b<d;b++){
            scanf("%d",&MatriksA[a][b]);
        }
    }
    printf("Matriks B\n");
    for(a=0;a<d;a++){
        for(b=0;b<d;b++){
            scanf("%d",&MatriksB[a][b]);
        }
    }
    for(a=0;a<d;a++){
        for(b=0;b<d;b++){
            for(c=0;c<d;c++){
                hasil=hasil+MatriksA[a][c]*MatriksB[c][b];
            }
            MatriksAXB[a][b]=hasil;
            hasil=0;
        }
    }
    printf("Matriks AXB\n");
    for(a=0;a<d;a++){
        for(b=0;b<d;b++){
            printf("%d ",MatriksAXB[a][b]);
        }
        printf("\n");
    }
}
